### ISSUE TITLE
[Request]:  Disable Microsoft To Do Syncing

### ISSUE THREAD

**Issue by johnblommers** [dgsasha/remembrance#48]
### Tell us the problem or your need

I'm very privacy aware and do not trust Microsoft with any of my information. There fore I don't find it appropriate that Reminders has no feature to disable attempting to sync with it.

I do not want Microsoft getting any traffic from my computer over the Microsoft To Do feature. Period.

### Describe the solution you'd like

Add an option to Reminders to disable syncing with Microsoft To Do.

### Describe alternatives you've considered

At present I really like the feature set of Reminders but won't use it until the Microsoft entanglement is resolved.

### Additional context

Thanks for reading this far. My wish for maximum privacy is important to me.

**Comment by alexlyee** [comment]
I completely agree, and I do not think this is an unusual or extreme request. In my opinion, this should be default.

**Comment by dgsasha** [comment]
Yeah I can do this. This was actually the default at one point but it was changed to make things simpler. I'll make it so the Microsoft To Do backend is disabled until the user signs into a Microsoft account.

**Comment by dgsasha** [comment]
This might be unusually difficult, will have to think about it more. Will try to include it in version 5.1 because my efforts are focused on the redesign and windows port at the moment.
